draft-ietf-rats-architecture.md

```diff
@@ -823,7 +834,7 @@ private key is only known inside the device's hardware.  Thus, when Evidence and
 are used together, an appraisal procedure can be conducted based on Appraisal Policies that may not be specific to the
 device instance, but merely specific to the manufacturer providing the Endorsement. For example,
 an Appraisal Policy might simply check that devices from a given manufacturer have information
-matching a set of known-good reference values, or an Appraisal Policy might have a set of more complex
+matching a set of known-good Reference Values, or an Appraisal Policy might have a set of more complex
```

Comment:
Remove "known-good", assuming "known-good" is mentioned as an AKA for Reference Values.

Reply:
Done